Dispatcher/Supporting Files/Utils.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Utils.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 03/05/2023.
+//
+
+import UIKit
+
+struct Utils {
+    
+    static func showErrorAlert(_ msg: String) -> UIAlertController {
```

Comment:
The function only returns an UIAlertController and doesn't actually display it on the screen so the func name should be accordingly